multi_map_it_test.go

```diff
@@ -0,0 +1,399 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package hazelcast_test
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	hz "github.com/hazelcast/hazelcast-go-client"
+	"github.com/hazelcast/hazelcast-go-client/internal/it"
+	"github.com/hazelcast/hazelcast-go-client/types"
+)
+
+func TestMultiMap_Put(t *testing.T) {
+	it.MultiMapTester(t, func(t *testing.T, m *hz.MultiMap) {
+		targetValue := "value1"
+		assert.False(t, !it.MustBool(m.Put(context.Background(), "key", targetValue)), "multi-map put failed")
```

Comment:
Let's create and store the context in a variable, to make the code more clear (unless separate context is required):
```
ctx := context.Background()
```
Since we are testing `m.Put` it would be better to separate these into several lines, so the usage is more apparent, like:
```
increased, err := m.Put(ctx, "key", targetValue)
if err != nil {
    t.Fatal(err)
}
if !assert.True(t, increased, "multi-map put failed")
```

Reply:
I agree that would be more readable. I will fix these, thank you